packages/ui/src/hooks/entities.test.tsx

```diff
@@ -162,69 +164,4 @@ describe('useEntities', () => {
 `,
     );
   });
-
-  describe('comments', () => {
```

Comment:
Would be possible to keep a `comments`-related test? maybe not checking the actual comments per se, but simply creating a resource from a `YAML` with comments, then calling `toString` and see the string is the same as the original one.